opentelekomcloud/resource_opentelekomcloud_lb_certificate_v2_test.go

```diff
@@ -149,6 +162,11 @@ i1YhgnQbn5E0hz55OLu5jvOkKQjPCW+8Kg==
 -----END CERTIFICATE-----
 EOT
 
+  lifecycle {
+	ignore_changes = [certificate, private_key, type]
+  }
```

Comment:
Are those `ignore_changes` required every time? Why? If they are empty in the server response, they should be not set during `read...`

Reply:
Folks, I cannot fix all unclean parts of the provider with one pull request.

The ignore_changes are needed at many places if you do a larger automation project for the OpenTelekomCloud provider.
Even some of your current acceptance tests fail in practice (on a real tenant) with a non empty plan.
E.g. I just had to fiy the lb_certifcate case wich was completely failing.

The reason is that OTC sometimes "trims" inputs in the backgroud. With this behavior, ignore_changes can only be avoided when also avoiding trimming in the input .tf, which is not possible in some cases.
For updates, ignore_changes is still even mandatory.   

tl;dr: Cleaning up ignore_changes is a larger refactoring, esp. for many test cases.

Reply:
I undestand, that changes after apply is a common problem for OTC provider. But putting other fields aside, `type` is the field of schema which you've added in this PR. What is the problem with this field?

Reply:
If you don´t set type, the followup read returns "server"
I´ve added "server" as default, but not run the test without the constraint yet. Could do that, but I am not sure that this works.

Reply:
It should work if you will add `Computed: true,` to the `type` field

Reply:
After trying, it fixes the problem for type field, but not for certificate or private_key